leaflet/index.d.ts

```diff
@@ -145,7 +145,7 @@ declare namespace L {
 
     export class LatLngBounds {
         constructor(southWest: LatLngExpression, northEast: LatLngExpression);
-        constructor(latlngs: LatLngBoundsLiteral);
+        constructor(latlngs?: LatLngBoundsLiteral);
```

Comment:
That's not a problem we can solve in type definition. The code allows no argument so type def should allow it.